src/proto/blocks.proto

```diff
@@ -140,11 +140,18 @@ message RegisterValidator {
   uint64 nonce = 2;
 }
 
+message MempoolMessage {
+  oneof mempool_message {
```

Comment:
Why is this a oneof?

Reply:
Ah, this is leftover from my original commit when I had validation messages gossipped as well.

I can remove, but I was wondering if it is simpler to expand it in the future like this?

Reply:
Hm leaving as oneof for extensibility seems reasonable